packaging/build_wheel.bat

```diff
@@ -0,0 +1,29 @@
+@echo on
+
+set VC_VERSION_LOWER=16
+set VC_VERSION_UPPER=17
+if "%VC_YEAR%" == "2017" (
+    set VC_VERSION_LOWER=15
+    set VC_VERSION_UPPER=16
+)
+
+for /f "usebackq tokens=*" %%i in (`"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" -legacy -products * -version [%VC_VERSION_LOWER%^,%VC_VERSION_UPPER%^) -property installationPath`) do (
```

Comment:
Is it possible to condense this as an extra `step` in the circleci config that runs only on windows wheel builds?

Seems like it'd be better to do that instead of having it completely shell out to a separate script.

Reply:
It is not easy to share an environment between Bash and CMD, especially when you want to activate the env in CMD and get the env back in Bash.

Reply:
Actually, I come up with an idea that we could create a utility to activate the env and do sth in CMD by passing arguments.